README.asciidoc

```diff
@@ -0,0 +1,17 @@
+Maven Base Docker Image
+-----------------------
+
+Building
+~~~~~~~~
+
+`docker build -t base-maven .`
+
+Example Usage
+~~~~~
+
+* `cd [workDir]`
+
+* `docker run --privileged=true -v {backtick}pwd{backtick}:/opt/jboss/myproject base-maven mvn clean install -f /opt/jboss/myproject`
```

Comment:
why is --privileged=true required here? we should require as low permissions as possible.